### DISCUSSION TITLE
Use without release?

### DISCUSSION THREAD

**Discussion by flemzord** [goreleaser/goreleaser#3950] [Q&A] [answered]
Hello 

We have been using Goreleaser to manage our releases for several months. 
However, we have a second process to build docker images and binaries for each commit on main or pull requests 
We would like to integrate this into Goreleaser and have only one tool 

Any idea how to do this?

Regards 
Maxence

**Comment by caarlos0** [comment]
if you don't want to publish the generated artifacts anywhere, check `goreleaser releaser --help`, specifically the `--snapshot` option

**Reply by flemzord** [reply]
The release mode with --snapshot works well. But I would need to push the binaries / Docker images to their registry. 
Today with --snapshot, it will fetch the last tag.

**Reply by caarlos0** [reply, marked as answer]
In that case, you'd probably would need to use the nightlies feature: https://goreleaser.com/customization/nightlies/

**Comment by tebruno99** [comment]
I keep a 2nd goreleaser config file.  $project_dir/.goreleaser.ecr.yml.

I then use this goreleaser config with
```
..
builds:
...
..
release:
 disable: true
dockers:
...
..
..
```

This allows us to do the build, build the docker images and upload them to AWS ECR without publishing to the GitHub Releases for this build.

We still release/build by tags though because thats how go versioning works best.
